src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlServerDnsAlias.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlServerDnsAlias
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+```
+Get-AzureRmSqlServerDnsAlias [-DnsAliasName <String>] -ServerName <String> [-ResourceGroupName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> $serverDNSAliases = Get-AzureRmSqlServerDNSAlias -ServerName server -ResourceGroupName rg
```

Comment:
@maboja-msft for the examples you have in your markdown files, please add the corresponding output the user will see when executing the command. This gives the user a better idea of what to expect from the object being returned to them from the cmdlet.